tests/std/tests/P0645R10_text_formatting_utf8/test.cpp

```diff
@@ -8,6 +8,10 @@
 
 #include "test_format_support.hpp"
 
+#pragma warning(disable : 5105) // macro expansion producing 'defined' has undefined behavior

```

Comment:
Since both tests have special-purpose `env.lst` files, this warning may as well be suppressed there.

Reply:
> it won't matter in the end anyways right? Since this hack doesn't work

Yep: after implementing a more hygienic testing method, I reverted this block from both tests and the changes to both `env.lst` files.